readme.md

```diff
@@ -385,90 +507,53 @@ interface SubmitTerms {
-to choose the DID for which the credential will be issued. Otherwise, the extension SHOULD NOT offer the choice,
-but still MUST get the user’s consent to use this DID.
+The extension used a temporary DID for the communication channel.
+Since this DID is only used for a single interaction, it is not possible for the attester to verify the identity of the claimer until this step.
+In order for the attester to hold a proof that the claimer agreed to the terms, the claimer MUST sign the `RequestAttestation` object using the DID specified in the credential subject (field `claim.credentialSubject.id`).
+The attester SHOULD store the signed `RequestAttestation` object in case it is needed for later dispute resolution.
 
-The chosen or confirmed DID URI will be submitted as the `owner` field of the `claim` in the `'request-attestation'` message.
+
+The chosen or confirmed DID URI will be submitted as the `subject` field of the `claim` in the `'request-attestation'` message.
 The attester MUST only issue a credential to this DID.
-The attester MAY reject the request if this DID is different from the `owner` in the previous `'submit-terms'` message.
+The attester MUST use the provided `salt` values for the proof of the credential.
+The attester MAY reject the request if this DID is different from the `subject` in the previous `'submit-terms'` message.
 
-If the `quote` was provided, and the user has entered the password to decrypt the private key for signing the request,
-the extension SHOULD temporarily cache either the password or the unencrypted private key,
-so that the user does not need to enter it again when the payment needs to be transferred.
 
-|||
-|-|-|
-| direction | `extension -> dApp` |
+|              |                         |
+| ------------ | ----------------------- |
+| direction    | `extension -> dApp`     |
 | message_type | `'request-attestation'` |
 
 ```typescript
 interface RequestAttestation {
-    credential: {
-        claim: {
-            /** Hash of the CType */
-            cTypeHash: string
-
-            /** contents of the proposed credential */
-            contents: object
-
-            /** DID URI to issue the credential for */
-            owner: string
-        }
-
-        /** mapping of hashes to nonces */
-        claimNonceMap: Record<string, string>
-
-        /** list of hashes */
-        claimHashes: string[]
-
-        /** optional ID of the DelegationNode of the attester to be used in the attestation */
-        delegationId?: string
-
-        /** optional array of credentials of the attester to include in the attestation
-        *  @link https://kiltprotocol.github.io/sdk-js/interfaces/types_src.ICredential.html */
-        legitimations: ICredential[]
-
-        /** root hash of the data above */
-        rootHash: string
-    },
-    /** quote agreement signed by the claimer */
-    quote?: IQuoteAgreement
+    claim: AgreedCredential
+    /** quote agreement signed by the claimer
```

Comment:
not signed, right?

Reply:
[not signed and also not the IQuote from the SDK but the Quote defined in this doc](8a291effde4d2dc59e32c2d4189a6c2b4fbf9db2)